helpers/clickClearRefinements.ts

```diff
@@ -5,11 +5,20 @@ declare namespace WebdriverIOAsync {
 }
 
 browser.addCommand('clickClearRefinements', async () => {
+  const oldUrl = await browser.getUrl();
   const clearButton = await browser.$(`.ais-ClearRefinements-button`);
   // Assures us that the element is in the viewport
   await clearButton.scrollIntoView();
 
   await clearButton.click();
 
-  return browser.waitForElement(`.ais-ClearRefinements-button--disabled`);
+  await browser.waitForElement(`.ais-ClearRefinements-button--disabled`);
+
+  // Changing the URL will also change the page element IDs in Internet Explorer
+  // Not waiting for the URL to be properly updated before continuing can make the next tests to fail
+  return browser.waitUntil(
```

Comment:
Is there any code actually expecting any return result from `clickClearRefinements`?
If not, what if we just go with `await browser.waitUntil()` without return statement?
It'd be easy to add lines after that if needed later.
And it kind of explicitly say this async function actually don't return anything meaningful other than a promise.

Reply:
Actually `clickClearRefinements` is not even used :laughing: 

But yes, we could do this for all helpers that does not return any value :+1: